### PR TITLE
Fix main for sequence classifier

### PR DESCRIPTION
**Pull request by kategimranova**

### CODE REVIEW

File: src/utils/common.py [hunk 1]
```diff
@@ -217,14 +217,15 @@ def forward(self, outputs: torch.Tensor, labels: torch.Tensor) -> torch.Tensor:
...
         self.pos_emb = nn.Embedding(max_len, hidden_size)
         self.register_buffer("positions", torch.arange(max_len, dtype=torch.long).reshape(1, -1))
+        self.dropout = nn.Dropout(p=dropout)
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
         assert isinstance(self.positions, torch.Tensor)
         pos_emb = self.pos_emb(self.positions[:, : x.shape[1]])  # [1, seq_len, hidden_size]
         x = x + pos_emb
-        return x
+        return self.dropout(x)
```
Comment (SpirinEgor) [inline src/utils/common.py:231]: Apply dropout in SequenceClassifier model, keep PositionalEncoding without changes
